quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/FeaturesDistTest.java

```diff
@@ -28,7 +28,7 @@
 @LegacyStore
 public class FeaturesDistTest {
 
-    private static final String PREVIEW_FEATURES_EXPECTED_LOG = "Preview features enabled: account3, admin-fine-grained-authz, client-secret-rotation, declarative-user-profile, dpop, multi-site, recovery-codes, scripts, token-exchange, update-email";
+    private static final String PREVIEW_FEATURES_EXPECTED_LOG = "Preview features enabled: account3:v1, admin-fine-grained-authz:v1, client-secret-rotation:v1, declarative-user-profile:v1, dpop:v1, multi-site:v1, recovery-codes:v1, scripts:v1, token-exchange:v1, update-email:v1";
```

Comment:
It looks weird to me; having the version appended to the feature, which has only one version. It makes sense to have it when multiple versions of the feature exist.

Reply:
This is the desired behavior from @stianst and @vmuzikar. I believe this is the right way to do it as well. The user will always be shown exactly what version is in use - there is nothing to document nor do the users have to know how many versions exist.

Reply:
@shawkins Ok, I respect the decision if you've agreed on it.